pandas-stubs/core/generic.pyi

```diff
@@ -165,7 +165,7 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
         chunksize: int | None = ...,
         dtype: DtypeArg | None = ...,
         method: Literal["multi"]
-        | Callable[[SQLTable, Any, list[str], Iterable], int | None]
+        | Callable[[SQLTable, Any, list[str], Iterable[Any]], int | None]
```

Comment:
After some comments elsewhere, can you change this to be `Iterable[list | tuple]` ?

Reply:
Not sure if it needs to be `Iterable[list[Any] | tuple[Any]` or `Iterable[list[Any] | tuple[Any, ...]`